cmd/root.go

```diff
@@ -62,13 +56,40 @@ func New(config Config) (*cobra.Command, error) {
 
 	var err error
 
+	f := &flag{}
```

Comment:
We define the root command first so that we can use the pointer to the config flag in the subcommands